recipes/birdy/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "birdy" %}
+{% set version = "v0.6.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bird-house/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: ece0aa45130b874ec623b763027e1ee5da125fdbf83a30cdb622ed1d9facd1be
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install . --no-deps -vv
+  entry_points:
+    - birdy = birdy.cli.run:cli
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
+    - six
+    - click
+    - lxml
+    - owslib >=0.18.0
+    - jinja2
+    - wrapt
+    - funcsigs
+    - boltons
+  run:
+    - python
```

You will need to add the restriction here as well
```suggestion
    - python >=3
```